src/ContentPage/index.js

```diff
@@ -53,7 +53,7 @@ const FAQRoutes = (
       />
       <Route
         exact
-        path={`${routeDefinitions.FAQ_ARTICLE}/:articleId`}
+        path={`${routeDefinitions.FAQ_ARTICLE}/:categoryId/:articleId`}
```

Comment:
I'm not sure this is a good idea. 

Articles can belong to several categories, so it's conceptually a bit confusing IMHO. Also some articles will be reached after more than one nested category screen and this route does not cover those cases.

What do you think? :slightly_smiling_face:

Reply:
The idea is the each article would have multiple URLs instead of a single URI. So the URL would denote not a unique identifier of the article, but a history state. (Coming to that article from that category)

The `undefined` thing does pose an issue though. Maybe categoryId could be changed to `parentId` and in case of search results, it'd have a value of `search`, otherwise it would have the `categoryId`.

Another alternative is inspecting `history.entries`. In that case, the URL wouldn't have any info about the category.

Reply:
>  So the URL would denote not a unique identifier of the article, but a history state

Great point! :) However if that URI is going to denote the history, how does the current implementation cover the case when an article has been entered after more than one category screen(eg. at the 2nd categories screen)?

Imho breadcrumbs should just be a reflection of the browser history, but I also like your idea of having state embedded in the URI somehow.

Reply:
It's @jaroslav-kubicek's idea :-P At first I also implemented with history.

The nested category case is solved, because categories have only one parent. (It's a tree if you don't count the articles). It can be directly fetched from graphql as already done on the category list screen